chia/cmds/wallet_funcs.py

```diff
@@ -1470,10 +1481,12 @@ async def mint_vc(
         vc_record, txs = await wallet_client.vc_mint(
             decode_puzzle_hash(ensure_valid_address(did, allowed_types={AddressType.DID}, config=config)),
             CMDTXConfigLoader().to_tx_config(units["chia"], config, fingerprint),
-            None
-            if target_address is None
-            else decode_puzzle_hash(
-                ensure_valid_address(target_address, allowed_types={AddressType.XCH}, config=config)
+            (
+                None
+                if target_address is None
+                else decode_puzzle_hash(
+                    ensure_valid_address(target_address, allowed_types={AddressType.XCH}, config=config)
+                )
```

Comment:
Again, I'll defer to CI here I guess but do you by chance have an explanation for why this is necessary?